linked_list.rb

```diff
@@ -8,27 +9,67 @@ def initialize
   end
 
   def add_first(value)
+    new_node = Node.new(value)
 
+    new_node.next = @head if !@head.nil?
```

Comment:
I don't think you need the `if` you could always assign `new_node.next = @head`, if it's `nil` then `.next` should be nil anyway.  

In fact you could do the entire method with:
```ruby
@head = Node.new(value, @head)
```